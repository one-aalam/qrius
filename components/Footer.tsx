import cx from 'classnames';
import styles from './utility.module.css';

type Props = {};

const Footer: React.FC<Props> = ({ children }) => <footer className={cx(styles.px8, styles.py4)}>{children}</footer>;

export default Footer;
