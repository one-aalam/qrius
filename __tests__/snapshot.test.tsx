import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

it('renders footer', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
