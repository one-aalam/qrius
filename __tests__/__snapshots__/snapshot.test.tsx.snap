// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer 1`] = `
<footer
  className="px8 py4"
/>
`;
